s3/s3.go

```diff
@@ -190,6 +191,12 @@ const (
 	maxSizeForCopy = 5 * 1024 * 1024 * 1024 // The maximum size of object we can COPY
 )
 
+// Globals
+var (
+	// Flags
+	ACL = pflag.StringP("s3-acl", "", "", "Canned ACL used when creating buckets and/or storing objects in S3")
```

Comment:
This should start with a lower case letter otherwise it is exported.  I'd call it s3ACL probably.